### PR TITLE
ITautomate Connector

### PR DESCRIPTION
**Pull request by ITautomate**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by v-EgorKozhadei** [comment]
Hi @ITautomate!

Thank you for your submission!

Please sign the CLA, this step is required. You can [find it on your PR page](https://github.com/microsoft/PowerPlatformConnectors/pull/1764#issuecomment-1165571257).
After that we'll be able to start the review process.

Please let me know if you have any question.

File: certified-connectors/ITautomate/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,272 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v1",
```
Comment (v-EgorKozhadei) [inline certified-connectors/ITautomate/apiDefinition.swagger.json:4]: @ITautomate please change the version to 1.0

File: certified-connectors/ITautomate/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CommandRunResultModel_X"
+            }
+          },
+          "default": {
+            "description": "OK",
+            "schema": {}
```
Comment (v-EgorKozhadei) [inline certified-connectors/ITautomate/apiDefinition.swagger.json:72]: @ITautomate empty schemas are not allowed. You can remove an empty schema for default response.

File: certified-connectors/ITautomate/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CommandRunResultModel_X"
+            }
+          },
+          "default": {
+            "description": "OK",
```
Comment (v-EgorKozhadei) [inline certified-connectors/ITautomate/apiDefinition.swagger.json:71]: @ITautomate you can change it to something like `Error` if you want, you have `OK` for 200 code and for default right now.

File: certified-connectors/ITautomate/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+            "schema": {
+              "type": "object"
+            }
+          },
+          "400": {
+            "description": "Invalid commandId"
+          },
+          "default": {
+            "description": "OK",
+            "schema": {}
```
Comment (v-EgorKozhadei) [inline certified-connectors/ITautomate/apiDefinition.swagger.json:110]: @ITautomate please remove an empty schema here as well. You can also change the description value here for the default response if you want.

File: certified-connectors/ITautomate/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
```
Comment (v-EgorKozhadei) [inline certified-connectors/ITautomate/apiDefinition.swagger.json:148]: @ITautomate please remove an empty schema here too.

File: certified-connectors/ITautomate/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CommandList"
+            }
+          },
+          "default": {
+            "description": "OK",
+            "schema": {}
```
Comment (v-EgorKozhadei) [inline certified-connectors/ITautomate/apiDefinition.swagger.json:174]: @ITautomate please remove an empty schema here.

File: certified-connectors/ITautomate/apiProperties.json [hunk 7]
```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "07c09c6f-e066-4fa5-8a49-dfe289854a2f",
```
Comment (v-EgorKozhadei) [inline certified-connectors/ITautomate/apiProperties.json:8]: @ITautomate you can remove this one, secret-look-a-like-values are not allowed. Please change it to just an empty string.

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @ITautomate! Please check comments after the review, thank you!

File: certified-connectors/ITautomate/apiDefinition.swagger.json (hunk 2 above)
Comment (ITautomate) [reply certified-connectors/ITautomate/apiDefinition.swagger.json:72]: Hi @v-EgorKozhadei,  so should we remove the schema property all together?

File: certified-connectors/ITautomate/apiProperties.json (hunk 7 above)
Comment (ITautomate) [reply certified-connectors/ITautomate/apiProperties.json]: Fixed

File: certified-connectors/ITautomate/apiDefinition.swagger.json (hunk 6 above)
Comment (ITautomate) [reply certified-connectors/ITautomate/apiDefinition.swagger.json]: Fixed

File: certified-connectors/ITautomate/apiDefinition.swagger.json (hunk 5 above)
Comment (ITautomate) [reply certified-connectors/ITautomate/apiDefinition.swagger.json]: Fixed

File: certified-connectors/ITautomate/apiDefinition.swagger.json (hunk 2 above)
Comment (ITautomate) [reply certified-connectors/ITautomate/apiDefinition.swagger.json]: Fixed

**Review by v-EgorKozhadei — APPROVED** [review]

**Pull request merged**